site-src/geps/gep-709.md

```diff
@@ -0,0 +1,260 @@
+# GEP-709: Cross Namespace References from Routes
+
+* Issue URL: https://github.com/kubernetes-sigs/gateway-api/issues/709
+* Status: Implementable
+
+## TLDR
+
+This GEP attempts to tackle both cross namespace forwarding and route inclusion.
```

Comment:
It doesn't seem like we are actually defining route inclusion in this GEP.

We should probably say that this is: prepare the API for route inclusion.

Reply:
BTW if there are references, it would be good to include to define what route inclusion means.

Reply:
Unfortunately we only have feature request issues that can be solved with Route inclusion (#634, #695) and a couple design docs. For now I've just clarified this sentence and added a brief description of Route inclusion.